utils/backups_test.go

```diff
@@ -0,0 +1,45 @@
+/*
+Copyright 2022.
```

Comment:
```suggestion
Copyright 2023.
```

And usually copyrights have also owner of right, in our case "Red Hat Inc."

Reply:
multiple occurences